utils/rpms/daos.spec

```diff
@@ -48,7 +39,7 @@ BuildRequires: libisa-l_crypto-devel
 BuildRequires: libisal-devel
 BuildRequires: libisal_crypto-devel
 %endif
-BuildRequires: raft-devel = 0.6.0
+BuildRequires: raft-devel >= 0.6.0
```

Comment:
This might be the only one that bites us.  We'll find out soon enough.  Although this should have all rolled back building to 15.1 where there was no _raft_ (AFAIU), so this should be fine.